docs/csharp/fundamentals/exceptions/creating-and-throwing-exceptions.md

```diff
@@ -26,8 +26,8 @@ Programmers should throw exceptions when one or more of the following conditions
 
   :::code language="csharp" source="snippets/exceptions/Program.cs" ID="InvalidArg":::
 
-> [!NOTE]
-> The example above is for illustrative purposes. Index validating via exceptions is in most cases bad practice. Exceptions should be reserved to guard against exceptional program conditions, not for argument checking as above.
+  > [!NOTE]
+  > The preceding example is for illustrative purposes. Ideally, you don't use exceptions to validate arguments, but to handle exceptional program conditions.
```

Comment:
I think we should reword this. Consider this change:

```suggestion
> [!NOTE]
> The preceding example shows how to use the `InnerException` property. It's intentionally simplified. In practice, you'd check that an index is in range before using it. You'd use this technique of wrapping an exception when a member of a parameter throws an exception you couldn't anticipate before calling the member.
```

Reply:
How about - to remove future tense:
  > The preceding example shows how to use the `InnerException` property. It's intentionally simplified. In practice, you should check that an index is in range before using it. You could use this technique of wrapping an exception when a member of a parameter throws an exception you couldn't anticipate before calling the member.

Reply:
I like it. I updated my suggestion to match your edits.